swc/private/swc.bzl

```diff
@@ -110,6 +111,14 @@ def _replace_ext(f, ext_map):
         return new_ext
     return None
 
+def _replace_extension(f, ext):
+    i = f.rfind(".")
+    return f if i <= 0 else f[:-(len(f) - i)] + ext
+
+def _remove_extension(f):
```

Comment:
Is this better than `_replace_extension(f, "")` ? You're just avoiding that one extra string concat but "append an empty string" might already be optimized by the language runtime

Reply:
I prefer a method with a descriptive name myself but it probably makes no difference at runtime like you're saying 🤷